concept/search_api.go

```diff
@@ -16,35 +16,35 @@ type SearchAPI interface {
 	GTG() error
 }
 
-type conceptSearchAPI struct {
+type internalConcordancesAPI struct {
 	endpoint   string
 	apiKey     string
 	httpClient *http.Client
 	batchSize  int
 }
 
 func NewSearchAPI(endpoint string, apiKey string, batchSize int) SearchAPI {
-	return &conceptSearchAPI{
+	return &internalConcordancesAPI{
 		endpoint:   endpoint,
 		apiKey:     apiKey,
 		httpClient: &http.Client{},
 		batchSize:  batchSize,
 	}
 }
 
-func (search *conceptSearchAPI) SearchConcepts(ctx context.Context, conceptIDs []string) (map[string]Concept, error) {
-	combinedResult := make(map[string]Concept)
+func (search *internalConcordancesAPI) SearchConcepts(ctx context.Context, conceptIDs []string) (map[string]Concept, error) {
 	tid, err := tidUtils.GetTransactionIDFromContext(ctx)
-
 	if err != nil {
 		tid = tidUtils.NewTransactionID()
 		log.WithField(tidUtils.TransactionIDKey, tid).
 			WithError(err).
-			Warn("Transaction ID error for requests of concepts to concept search API: Generated a new transaction ID")
+			Info("No Transaction ID provided for concept request, generating a new transaction id")
```

Comment:
OCD says we should be consistent in the capitalisation of Transaction ID, at least within the scope of a single log message.